Formula/libgusb.rb

```diff
@@ -0,0 +1,94 @@
+class Libgusb < Formula
+  desc "GObject wrappers for libusb1"
+  homepage "https://github.com/hughsie/libgusb"
+  url "https://people.freedesktop.org/~hughsient/releases/libgusb-0.3.0.tar.xz"
+  sha256 "d8e7950f99b6ae4c3e9b8c65f3692b9635289e6cff8de40c4af41b2e9b348edc"
+
+  depends_on "gobject-introspection" => :build
+  depends_on "meson-internal" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python" => :build
+  depends_on "vala" => :build
+  depends_on "glib"
+  depends_on "libusb"
+
+  # this file changes often, so the checksum will need to be updated often as well
```

Comment:
I don't see that working for us, really… it's gonna end up being a pain to maintain, will cause from-source builds to fail. Is there no place where this resource is versioned?

Reply:
Hmmm... shall I put the latest version in formula-patches?

Reply:
Or `brew mirror`